### ISSUE TITLE
ALL losses goes down at first and then keep rising.

### ISSUE THREAD

**Issue by supersyz** [fvmassoli/cross-resolution-face-recognition#5]
Hi,thank you for your extraordinary job.
I  made the validation set by moving 10 images of each id in training set  to the directory of the same id name.
Thus the validation set has the same number of directories of training set,and each directory has 10 images.
But when I tried to run the code with default hyper parameters,It didn't converge with very large losses.
So I made some adjustment by switching optimizer to 

`optimizer = Adam(params=sm.parameters(),
lr=args.learning_rate,weight_decay=3e-04)`

and initial learning rate to 5e-4.

All the losses went down soon.The valid loss HR decreased to around 0.5 with  Valid acc HR 95%,the valid loss LR 4.5 with Valid acc LR around 30%,and the training LR loss 1.12 with training LR acc  around 88%.
but after reaching the bottom,all losses stopped decreasing. The valid loss began to slightly grow and the training loss grow rapidly  even close to the first training loss.

What could be reason for this?
Thx a lot.

**Comment by fvmassoli** [comment]
Perhaps you should set a policy to decrease the learning rate or use a lower learning rate. Moreover, in my experience SGD worked better than Adam. 
Anyhow, it does not seem to be a bug of the code.

**Comment by fvmassoli** [comment]
I am closing the issue. In case you can write in the "discussion" section
